docs/how-to/contribute/code.rst

```diff
@@ -8,336 +8,262 @@ If you experience problems with Toga, `log them on GitHub`_. If you want to
 contribute code, please `fork the code`_ and `submit a pull request`_.
 
 .. _log them on GitHub: https://github.com/beeware/toga/issues
-.. _fork the code: https://github.com/beeware/toga
+.. _fork the code: https://github.com/beeware/toga/fork
 .. _submit a pull request: https://github.com/beeware/toga/pulls
 
-.. _setup-dev-environment:
+.. _dev-environment-prereqs:
 
-Set up your development environment
-===================================
+Prerequisites
+=============
 
-First, ensure that you have Python 3 and pip installed. To do this, run:
+You'll need to install the following prerequisites.
 
 .. tabs::
 
   .. group-tab:: macOS
 
-    .. code-block:: console
-
-      $ python3 --version
-      $ pip3 --version
+    View the :ref:`macOS prerequisites <macos-prerequisites>`.
 
   .. group-tab:: Linux
 
-    .. code-block:: console
-
-      $ python3 --version
-      $ pip3 --version
+    View the :ref:`Linux prerequisites <linux-prerequisites>`.
 
   .. group-tab:: Windows
 
-    .. code-block:: doscon
+    View the :ref:`Windows prerequisites <windows-prerequisites>`.
 
-      C:\...>python3 --version
-      C:\...>pip3 --version
+.. _dev-environment-tldr:
 
-Create a virtual environment
-----------------------------
+:spelling:ignore:`tl;dr` - Dev Quick-Setup
+==========================================
 
-The recommended way of setting up your development environment for Toga
-is to install a virtual environment, install the required dependencies and
-start coding. To set up a virtual environment, run:
+Set up the dev environment by running:
 
 .. tabs::
 
   .. group-tab:: macOS
 
     .. code-block:: console
 
-      $ python3 -m venv venv
-      $ source venv/bin/activate
+      $ git clone https://github.com/beeware/toga.git
+      $ cd toga
+      $ python -m venv .venv
```

Comment:
On macOS and Linux, the `-m venv` command must use `python3`, because `python` may point at Python 2, or not exist at all.

Once inside the virtual environment, `python` is fine for all platforms.

This should be fixed in the Briefcase docs as well.

Reply:
@mhsmith Fixes in. 

Please see: https://github.com/beeware/toga/pull/3605 and https://github.com/beeware/briefcase/pull/2367.